packages/ruleset-migrator/src/tree/index.ts

```diff
@@ -54,7 +54,7 @@ export class Tree {
       }
 
       const identifier = Tree.identifier(specifier, scope);
-      existingImportDeclaration.push({ imported: identifier, local: identifier, default: _default });
+      existingImportDeclaration.push({ imported: b.identifier(specifier), local: identifier, default: _default });
```

Comment:
We already do it here https://github.com/stoplightio/spectral/blob/f524a8c0a6744127dd878268ac8f6a0d1fc4f889/packages/ruleset-migrator/src/tree/index.ts#L46, but this spot was missed.